cmd/torrent_remove.go

```diff
@@ -66,6 +66,9 @@ func RunTorrentRemove() *cobra.Command {
 		options := qbittorrent.TorrentFilterOptions{}
 		if filter != "" {
 			options.Filter = qbittorrent.TorrentFilter(filter)
+			if len(includeCategory) == 0 {
+				includeCategory = []string{""}
+			}
```

Comment:
Why does it require to include an empty string as category for the list? By default if no category is passed it should return everything. Unless the options set it to an empty array since it is instantiated on line 38 and qBit counts that as none :thinking:

Reply:
If `len(includeCategory) -- 0`, `GetTorrentsCtx()` is never called if, for example, only filtering is used.